fsm/client.go

```diff
@@ -251,6 +252,37 @@ func (c *client) FindAll(input *FindInput) (output *FindOutput, err error) {
 	return NewFinder(c.f.Domain, c.c).FindAll(input)
 }
 
+func (c *client) FindAllWalk(input *FindInput, workflowInfosFunc WorkflowInfosFunc) error {
```

Comment:
I modeled the signature of this function after WalkOpenWorkflows, but it’s a little weird that the workflowInfosFunc takes a slice of infos instead of just a single info. It’s means callers still have to have their own looping. Can anyone remember why we (i.e. I) did it like this? If we don’t have a good reason, I’m thinking we should just make it a singular walk func and combine these two together. Thoughts?

Reply:
I don't remember